pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+target-version = ["py38", "py39", "py310", "py311"]
+
+[tool.isort]
+profile = "black"
+multi_line_output = 3
+
+[tool.mypy]
+#check_untyped_defs = true
```

Comment:
To be addressed later on, or a leftover?

Reply:
Oh, it's used actually :) Let me fix it